modules/project-factory/automation.tf

```diff
@@ -58,8 +58,13 @@ module "automation-bucket" {
     for k, v in lookup(each.value, "iam_bindings", {}) : k => merge(v, {
       members = [
         for vv in v.members : try(
+          # rw (infer local project and automation prefix)
+          module.automation-service-accounts["${each.key}/automation/${vv}"].iam_email,
+          # automation/rw or sa (infer local project)
           module.automation-service-accounts["${each.key}/${vv}"].iam_email,
+          # project/automation/rw project/sa
           var.factories_config.context.iam_principals[vv],
+          # fully specified principal
           vv
```

Comment:
Should we do the `strcontains` thing here too?

Reply:
done, good call